nifi-bootstrap/pom.xml

```diff
@@ -34,13 +34,26 @@ language governing permissions and limitations under the License. -->
             <artifactId>nifi-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-security-utils</artifactId>
+        </dependency>
+        <dependency>
             <groupId>javax.mail</groupId>
             <artifactId>mail</artifactId>
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-expression-language</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
```

Comment:
I checked the licensing and indeed it is apache with no claimed copyrights and no declared notices.  Good.  However, there are several dependencies.  Did you verify none of them require any updates to our license/notices for the convenience binary/assembly?